### PR TITLE
CI Fixes 2023-12-02

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Reverts #23450

Just makes it so we dont even run these in CI

## Why It's Good For The Game
CI should work

### CODE REVIEW

File: code/game/world.dm [hunk 1]
```diff
@@ -292,11 +292,13 @@ GLOBAL_LIST_EMPTY(world_topic_handlers)
 	F << GLOB.log_directory
 
 /world/Del()
+	// Calling these cleanups causes issues in CI
+	// However in CI, we dont care because the process is being shot anyway
+	#ifdef UNIT_TESTS
```
Comment (AffectedArc07) [inline code/game/world.dm:297]:
```suggestion
	#ifndef UNIT_TESTS
```

**Review by S34NW — APPROVED** [review]
Our code is very good and robust

**Comment by AffectedArc07** [comment]
image
```
Recent proc calls:
/world/Del
/datum/test_runner/proc/Finalize
/datum/controller/subsystem/proc/ignite
/datum/controller/subsystem/ticker/fire
/datum/timedevent/proc/getcallingtype
/proc/bitfield2list
/datum/timedevent/proc/bucketJoin
/datum/timedevent/New
```

**Comment by S34NW** [comment]
Fixed in https://github.com/ParadiseSS13/Paradise/pull/23501

**Pull request closed without merging**